tests/lib/fixture.py

```diff
@@ -5,41 +5,63 @@
 from __future__ import annotations
 
 from os import environ
-from typing import Callable
-from unittest.mock import MagicMock, create_autospec
+from typing import Callable, Iterator
+from unittest.mock import patch
 
 import pytest
-from aioeapi import Device
 from click.testing import CliRunner
 
-from anta.device import AntaDevice
+from anta.device import AntaDevice, AsyncEOSDevice
 from anta.inventory import AntaInventory
+from anta.models import AntaCommand
 from anta.result_manager import ResultManager
 from anta.result_manager.models import TestResult
 from tests.lib.utils import default_anta_env
 
+DEVICE_HW_MODEL = "pytest"
+DEVICE_NAME = "pytest"
+COMMAND_OUTPUT = "retrieved"
+
+
+@pytest.fixture
+def device(request: pytest.FixtureRequest) -> Iterator[AntaDevice]:
```

Comment:
I don't know if we should not use a different name for the fixture(s) to make it clear what this is